sonic-chassisd/scripts/chassisd

```diff
@@ -559,6 +612,161 @@ class ModuleUpdater(logger.Logger):
                         self._cleanup_chassis_app_db(module)
                     self.down_modules[module]['cleaned'] = True
 
+#
+# Module Updater ==============================================================
+#
+
+
+class SmartSwitchModuleUpdater(ModuleUpdater):
+
+    def __init__(self, log_identifier, chassis, my_slot, supervisor_slot):
```

Comment:
@rameshraghupath not sure what is the use of slot in SMART switch context. can we remove?

Reply:
> @rameshraghupath not sure what is the use of slot in SMART switch context. can we remove?

Resolved
@prgeor